server/modules/instantiation/jobs.js

```diff
@@ -88,6 +88,13 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
         }
     }
 
+    class RunAfterBuildTaskJob extends ComponentInstanceJob {
```

Comment:
jobs.js seems like a candidate for a separate directory/namespace/package
please create one and move all classes from this file to separate files